### DISCUSSION TITLE
2592FCEC-29C1-4A4B-ADE9-CAD4594EE5AC

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#117] [General]
# Blue Venus

This add-on modifies Celestia so that Blue Venus is displayed in place of the actual Venus textures. This addon was also updated in 2022 to feature a MIE defined atmosphere over its original atmosphere.

https://celestia.mobi/addon?item=2592FCEC-29C1-4A4B-ADE9-CAD4594EE5AC&lang=zh_CN

**Comment by gtfggjbdrff** [comment]
地球化金星

**Reply by zjx29rjhdh-pixel** [reply]
IT ISN’T WORKING FOR ME NOW

**Reply by rhiandevera64-design** [reply]
Fun Fact: Venus Is Used To Be Hatible
